cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc

```diff
@@ -47,23 +47,23 @@ Databases created after this command execution will also be associated with thes
 | Command | Description
 
 | [source, cypher]
-GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname \| *}} TO role[, ...]
+GRANT database-privilege ON {DEFAULT DATABASE \| DATABASE[S] {dbname [, ...] \| *}} TO role[, ...]
```

Comment:
Is there a reason for using **dbname** here? The examples above all say just **name.**

Reply:
That's just how it was, I will change it to make it consistant